### PR TITLE
Add 'Has Workspace' mode to show Linear issues linked to local worktrees

### PR DESCRIPTION
**Pull request by AmethystLiang**
## Summary

Introduces a new Linear task view mode—**'Has Workspace'**—that displays only Linear issues already linked to local Orca worktrees or folder workspaces. Users can quickly open an attached workspace from the issue drawer or task list, or start a new one with a fallback option.

### Core Feature
- New `linearMode: 'in-orca'` view that collects linked Linear identifiers from all worktrees and fetches their current issue state
- Issue attachment index for efficient per-row workspace lookups (one pass per list render, not per row)
- Deduplication logic that normalizes identifiers (case-insensitive, URL-parsed) and handles multi-workspace scenarios
- Workspace-scoped matching: respects both explicit `linkedLinearIssueWorkspaceId` and inferred workspace scope from provider context

### UI & UX
- **Issue drawer**: Shows attached workspace with 'Open workspace' primary button; dropdown offers 'Start new workspace' fallback
- **Task list rows**: Icon changes from `→` (start) to `📁` (open) when workspace is attached; shows workspace name inline
- **Search & filter**: In 'Has Workspace' mode, search queries filter by identifier, title, team, and assignee (not a Linear API query)
- **Error states**: Partial load failures show recoverable message; empty state guides users to start from a Linear issue

### Localization
Added ~12 translation keys per language (en, es, ja, ko, zh) for the new mode, button labels, empty states, and error messages.

### Implementation Details
- `task-page-linear-in-orca-issues.ts`: Collects refs from worktrees with dedup, filters, and concurrent read pooling (6 concurrent fetches by default)
- `linear-issue-workspace-attachment.ts`: Finds and indexes workspace attachments; handles scope matching (workspace ID, org URL key, provider context)
- `linear-issue-workspace-open.ts`: Opens attached workspace via `activateAndRevealWorktree`/`activateAndRevealFolderWorkspace`, or calls fallback to start a new one
- `worktree-attachment-label.ts`: Shared label generation (display name → branch name → path tail) for consistent labeling across GitHub, Linear, and future work-item surfaces

## Testing
- Unit tests cover deduplication, normalization, filtering, workspace scope matching, and index lookups
- Tests verify both exact and legacy (unscoped) identifier links, archived worktree exclusion, and concurrent read limiting
- Edge cases: cross-workspace collisions, organization key mismatches, URL parsing, multi-locale labels

## Verification
- No visual change—follows STYLEGUIDE.md tokens and button sizing
- Uses existing store selectors (`useAllWorktrees`) and activation utilities
- Respects folder workspace and git worktree use cases; compatible with SSH execution hosts
- Cross-platform: no platform-specific paths or shortcuts in new code

### CODE REVIEW

**Comment by AmethystLiang** [comment]
@coderabbitai Fixing. Will be in the next commit

File: src/renderer/src/components/TaskPage.tsx [hunk 1]
```diff
@@ -5196,16 +5215,50 @@ export default function TaskPage(): React.JSX.Element {
...
-    return displayedLinearIssues.filter((issue) => linearTeamSelection.has(issue.team.id))
-  }, [activeLinearIssueContextLabel, displayedLinearIssues, linearTeamSelection])
+    return searchedIssues.filter((issue) => linearTeamSelection.has(issue.team.id))
+  }, [
+    activeLinearIssueContextLabel,
+    appliedLinearSearch,
+    displayedLinearIssues,
+    linearMode,
+    linearTeamSelection
+  ])
```
[the remark this answers is not in the document]
Comment (AmethystLiang) [reply src/renderer/src/components/TaskPage.tsx:5266]: Fixing. Will be in the next commit

File: src/renderer/src/components/LinearIssueWorkspace.tsx [hunk 2]
```diff
@@ -949,6 +1029,29 @@ export default function LinearIssueWorkspace({
...
+              <div className="p-3">
+                {attachedWorkspaceLabel ? (
+                  <button
+                    type="button"
+                    onClick={handleOpenOrUseIssue}
+                    className="flex min-h-9 w-full min-w-0 items-center gap-2 rounded-md px-2 py-1.5 text-left text-sm text-muted-foreground transition hover:bg-accent hover:text-accent-foreground focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring"
+                  >
+                    <FolderOpen className="size-4 shrink-0" />
+                    <span className="min-w-0 flex-1 truncate">{attachedWorkspaceLabel}</span>
+                  </button>
```
[the remark this answers is not in the document]
Comment (AmethystLiang) [reply src/renderer/src/components/LinearIssueWorkspace.tsx:1051]: Fixing. Will be in the next commit

File: src/renderer/src/i18n/locales/es.json [hunk 3]
```diff
@@ -302,6 +302,15 @@
...
     "lib": {
+      "linear": {
+        "issue": {
+          "workspace": {
+            "open": {
+              "4f2c1d8a3b": "No se pudo abrir el espacio de trabajo vinculado a este issue."
+            }
+          }
+        }
+      },
```
[the remark this answers is not in the document]
Comment (AmethystLiang) [reply src/renderer/src/i18n/locales/es.json]: Fixing. Will be in the next commit

File: src/renderer/src/components/TaskPage.tsx [hunk 4]
```diff
@@ -8608,18 +8772,41 @@ export default function TaskPage(): React.JSX.Element {
...
+                                    className={buttonClassName}
+                                  >
+                                    {mode.label}
+                                  </button>
+                                </TooltipTrigger>
+                                <TooltipContent side="bottom" sideOffset={6}>
+                                  {translate(
+                                    'auto.components.TaskPage.linearModeHasWorktreeTooltip',
+                                    'Linear tickets linked to a local Orca worktree'
+                                  )}
```
[the remark this answers is not in the document]
Comment (AmethystLiang) [reply src/renderer/src/components/TaskPage.tsx:8803]: Fixing. Will be in the next commit

File: src/renderer/src/i18n/locales/ja.json [hunk 5]
```diff
@@ -302,6 +302,15 @@
...
     "lib": {
+      "linear": {
+        "issue": {
+          "workspace": {
+            "open": {
+              "4f2c1d8a3b": "このイシューにリンクされたワークスペースを開けませんでした。"
+            }
+          }
+        }
+      },
```
[the remark this answers is not in the document]
Comment (AmethystLiang) [reply src/renderer/src/i18n/locales/ja.json]: Fixing. Will be in the next commit

File: src/renderer/src/i18n/locales/zh.json [hunk 6]
```diff
@@ -302,6 +302,15 @@
...
     "lib": {
+      "linear": {
+        "issue": {
+          "workspace": {
+            "open": {
+              "4f2c1d8a3b": "无法打开关联到此议题的工作区。"
+            }
+          }
+        }
+      },
```
[the remark this answers is not in the document]
Comment (AmethystLiang) [reply src/renderer/src/i18n/locales/zh.json]: Fixing. Will be in the next commit
